transmart-copy/src/test/resources/examples/SURVEY_INC/part1/i2b2metadata/i2b2_secure.tsv

```diff
@@ -0,0 +1,4 @@
+c_hlevel	c_fullname	c_name	c_synonym_cd	c_visualattributes	c_totalnum	c_basecode	c_metadataxml	c_facttablecolumn	c_tablename	c_columnname	c_columndatatype	c_operator	c_dimcode	c_comment	c_tooltip	m_applied_path	update_date	download_date	import_date	sourcesystem_cd	valuetype_cd	m_exclusion_cd	c_path	c_symbol	i2b2_id	secure_obj_token
+3	\Projects\Survey_Inc\part1\Demographics\Gender\	Gender	N	LAC		gender		CONCEPT_CD	CONCEPT_DIMENSION	CONCEPT_PATH	T	LIKE	\Demographics\Gender\			@				SURVEY_INC						SURVEY_INC
```

Comment:
This works for the tests, but it wouldn't show up in the tree in TranSMART due to missing parent nodes if we would like to use this test data set for other purposes.